Lib/site.py

```diff
@@ -390,6 +390,22 @@ def setcopyright():
         files, dirs)
 
 
+def _register_detect_pip_usage_in_repl():
+    old_excepthook = sys.excepthook
+
+    def detect_pip_usage_in_repl(typ, value, traceback):
+        old_excepthook(typ, value, traceback)
+        if typ is SyntaxError and (
+            "pip install" in value.text or "pip3 install" in value.text
+        ):
+            print("\nThe Python package manager (pip) can only be used from"
```

Comment:
I have doubts about using `print()`. Are there any precedences? Wouldn't be better to modify the text of the SyntaxError?

Reply:
Good point. I've fixed this to set `exc.msg`

The new output is:
```
$ python -i -c "pip install qwerty"
  File "<string>", line 1
    pip install qwerty
              ^
SyntaxError: The Python package manager (pip) can only be used from outside of Python.
Please try the `pip` command in a separate terminal or command prompt.
>>>
```